sql/moz-fx-data-experiments/monitoring/experimenter_experiments_v1/query.py

```diff
@@ -0,0 +1,211 @@
+    is_control: bool
+    slug: str
+    ratio: int
+
+
+@attr.s(auto_attribs=True)
+class ExperimentV1:
+    """Experimenter v1 experiment."""
+
+    slug: str  # experimenter slug
+    type: str
+    status: str
+    start_date: Optional[datetime.datetime]
+    end_date: Optional[datetime.datetime]
+    variants: List[Variant]
+    proposed_enrollment: Optional[int] = attr.ib(converter=_coerce_none_to_zero)
+    normandy_slug: Optional[str] = None
+    is_high_population: Optional[bool] = None
+
+    @staticmethod
+    def _unix_millis_to_datetime(num: Optional[float]) -> Optional[datetime.datetime]:
+        if num is None:
+            return None
+        return datetime.datetime.fromtimestamp(num / 1e3, pytz.utc)
+
+    @classmethod
+    def from_dict(cls, d) -> "ExperimentV1":
+        converter = cattr.Converter()
+        converter.register_structure_hook(
+            datetime.datetime,
+            lambda num, _: cls._unix_millis_to_datetime(num),
+        )
+        return converter.structure(d, cls)
+
+    def to_experiment(self) -> "Experiment":
+        """Convert to Experiment."""
+        control_slug = None
+
+        control_slugs = [
+            variant.slug for variant in self.variants if variant.is_control
+        ]
+        if len(control_slugs) == 1:
+            control_slug = control_slugs[0]
+
+        return Experiment(
+            normandy_slug=self.normandy_slug,
+            experimenter_slug=self.slug,
+            type=self.type,
+            status=self.status,
+            start_date=self.start_date,
+            end_date=self.end_date,
+            proposed_enrollment=self.proposed_enrollment,
+            reference_branch=control_slug,
+            is_high_population=self.is_high_population or False,
+        )
+
+
+@attr.s(auto_attribs=True)
+class ExperimentV4:
+    """Represents a v4 experiment from Experimenter."""
+
+    slug: str  # Normandy slug
+    active: bool
+    startDate: Optional[datetime.datetime]
+    endDate: Optional[datetime.datetime]
+    proposedEnrollment: int
+    referenceBranch: Optional[str]
+
+    @classmethod
+    def from_dict(cls, d) -> "ExperimentV4":
+        converter = cattr.Converter()
+        converter.register_structure_hook(
+            datetime.datetime,
+            lambda num, _: pytz.utc.localize(
+                datetime.datetime.strptime(num, "%Y-%m-%d")
+            ),
+        )
+        return converter.structure(d, cls)
+
+    def to_experiment(self) -> "Experiment":
+        """Convert to Experiment."""
+        return Experiment(
+            normandy_slug=self.slug,
+            experimenter_slug=None,
+            type="v4",
+            status="Live" if self.active else "Complete",
+            start_date=self.startDate,
+            end_date=self.endDate,
+            proposed_enrollment=self.proposedEnrollment,
+            reference_branch=self.referenceBranch,
+            is_high_population=False,
+        )
+
+
+def get_experiments():
```

Comment:
```suggestion
def get_experiments() -> List[Experiment]:
```

may as well annotate if we're gonna annotate :)